### ISSUE TITLE
Why the feature has BatchNorm1d?

### ISSUE THREAD

**Issue by guozhiyao** [deepinsight/insightface#1654]
https://github.com/deepinsight/insightface/blob/66739ca6b8d053679f0434fd20d2c308ac487cb9/recognition/arcface_torch/backbones/iresnet2060.py#L164
Why the feature has BatchNorm1d? And set the weight.requires_grad=False?

**Comment by guozhiyao** [comment]
> same as arcface_mxnet

I want to know why this BN needs to be added, and whether it will affect performance if it is removed?
